src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/LeftCurlyCheck.java

```diff
@@ -151,7 +151,7 @@ public class LeftCurlyCheck
 
     /**
      * Specify the policy on placement of a left curly brace (<code>'{'</code>).
-     * */
+     */
```

Comment:
Why does this check care about extra *?

Reply:
The check detected this in the older implementation due to some bug, now the current implementation has fixed that bug. I thought it would be okay to remove this leading asterisk since it is unnecessary to have it

Reply:
Ok, please make sure to add this to our test cases in your new check PR